docs/kyma/04-03-cluster-installation.md

```diff
@@ -11,14 +11,42 @@ If you need to use Helm and access Tiller, complete the [additional configuratio
 
 >**CAUTION:** These instructions are valid starting with Kyma 1.2. If you want to install older releases, refer to the respective documentation versions.
 
-Choose your cloud provider and get started:
+Choose the installation type and get started:
 
-<div tabs>
+<div tabs name="provider-installation">
   <details>
   <summary>
-  GKE
+  GCP Marketplace
   </summary>
 
+  1. Access the Google Cloud Platform (GCP) Console at `https://console.cloud.google.com/`.
+  2. Select your project from the platform bar at the top of the page.
+  3. Search for GCP Marketplace in the search box and open the GCP Marketplace page.
```

Comment:
If you want to search then you need to search for Marketplace without GCP

Reply:
I changed the steps a bit